pkg/test/framework/components/istio/kube.go

```diff
@@ -622,6 +622,7 @@ func commonInstallArgs(ctx resource.Context, cfg Config, c cluster.Cluster, defa
 
 	if ctx.Settings().EnableDualStack {
 		args.AppendSet("values.pilot.env.ISTIO_DUAL_STACK", "true")
+		args.AppendSet("values.pilot.ipFamilyPolicy", string(corev1.IPFamilyPolicyRequireDualStack))
```

Comment:
You will also need to add `ipFamilies: [IPv4,IPv6]`

Reply:
Its an optional field. Normally its used when we want to define the order of IP families for a dual-stack service. I verified that setting the `ipFamilyPolicy` to `RequireDualStack` was indeed configuring the `istiod` service as a dual-stack service with `ipFamilies: [IPv4,IPv6]`